Makefile

```diff
@@ -46,6 +90,22 @@ DATA_DIR ?= $(PREFIX)/share/sameboy/
 FREEDESKTOP ?= true
 endif
 
+# Autoconf-style conventionally named variables.
+prefix ?= $(PREFIX)
+exec_prefix ?= $(prefix)
+includedir ?= $(prefix)/include
+bindir ?= $(exec_prefix)/bin
+libdir ?= $(exec_prefix)/lib
+datadir ?= $(prefix)/share
```

Comment:
this should probably be affected by or affect `DATA_DIR`

Reply:
Note that `DATA_DIR` is for that SameBoy specific data files like the bios while `datadir` is for package specific data files like the icons and desktop file. In autotools this is the difference between `datadir` and `datarootdir`.

Reply:
sure, I think the Makefile should have those two variables here as well.
Also, since a specific naming convention was already established with `DATA_DIR` and co., should these also follow the naming scheme?

Reply:
I think the Autotools convention is widely understood for packagers, so makes sense to follow.  I was a bit confused that DATA_DIR meant something different than datadir for SameBoy at first.

Reply:
Using the autotools conventions you are using `datadir` as if it was `datarootdir` and `DATA_DIR` is what `datadir` would be.

Reply:
Oh! I had missed that. Then I guess I should rename the `datadir` I used to `datarootdir` and have a new `datadir` be an alias to `DATA_DIR`.